pkgs/tools/networking/proxychains-ng/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+let
+  proxychains-ng-src-4-14 = fetchFromGitHub {
+    owner = "rofl0r";
+    repo = "proxychains-ng";
+    rev = "e895fb713ac1c1e92ee0795645f220573181c6fb";
+    sha256 = "03wk2xpxwc7kwlq6z9jf9df1vlh6p0dn0kf826fh1k7nfaa8c4py";
+  };
+in
+stdenv.mkDerivation rec {
+  name = "proxychains-ng-${version}";
+  version = "4.14";
+
+  src = proxychains-ng-src-4-14;
```

```suggestion
  src = fetchFromGitHub {
    owner = "rofl0r";
    repo = "proxychains-ng";
    rev = "e895fb713ac1c1e92ee0795645f220573181c6fb";
    sha256 = "03wk2xpxwc7kwlq6z9jf9df1vlh6p0dn0kf826fh1k7nfaa8c4py";
  };
```

Looks much more idomatic.